src/main/java/dao/CategoriesDao.java

```diff
@@ -0,0 +1,9 @@
+package dao;
+
+import java.util.List;
+
+import entity.Categories;
+
+public interface CategoriesDao {
+	List<Categories> getAllCategories();
```

Comment:
Tên hàm `getAllCategories()` cũng được, nhưng vì đây đã là CategoriesDao thì chỉ cần thiết kế tên hàm `findAll()` cũng hiểu là lấy hết categories rồi ha.
Ngoài ra, những DAO khác thì đang dùng findAll mà chỗ này là getAll thì chưa thống nhất style đặt tên -> cần thống nhất cách đặt tên hàm/biến trong toàn dự án.